### DISCUSSION TITLE
题目乱码，怎么解决呀，各位大佬

### DISCUSSION THREAD

**Discussion by JJJJJackson** [shuzijun/leetcode-editor#619] [Q&A]
![image](https://user-images.githubusercontent.com/96568427/221518013-03f83bdd-bd3a-401d-a093-9b18e6c68175.png)

**Comment by shuzijun** [comment]
https://github.com/shuzijun/leetcode-editor/issues/53

**Reply by JJJJJackson** [reply]
感谢，已解决

**Comment by FxCXQD** [comment]
感谢，解决了！
